pkg/lightning/backend/backend_test.go

```diff
@@ -273,6 +273,7 @@ func (s *backendSuite) TestImportFailedRecovered(c *C) {
 	c.Assert(err, IsNil)
 }
 
+//nolint:interfacer // change test case signature might cause Check failed to find this function?
```

Comment:
This should be a false positive case, the comment should be removed.

`check`'s finding rule is the same with go-test, only need to func name start with `Test`

Reply:
```
[2021-03-09T02:40:29.984Z] ----------------------------------------------------------------------
[2021-03-09T02:40:29.984Z] PANIC: backend_test.go:276: backendSuite.TestClose
[2021-03-09T02:40:29.984Z] 
[2021-03-09T02:40:29.984Z] ... Panic: backendSuite.TestClose argument should be *check.C
[2021-03-09T02:40:29.984Z] OOPS: 53 passed, 1 PANICKED
[2021-03-09T02:40:29.984Z] --- FAIL: TestKV (0.60s)
```

Oops...